summertime/model/single_doc/multilingual/mt5_model.py

```diff
@@ -0,0 +1,174 @@
+from transformers import MT5ForConditionalGeneration, MT5Tokenizer
+from .base_multilingual_model import MultilingualSummModel
+
+
+class MT5Model(MultilingualSummModel):
+    """
+    MT5 Model for Multilingual Summarization
+    """
+
+    # static variables
+    model_name = "mT5"
+    is_extractive = False
+    is_neural = True
+    is_multilingual = True
+
+    lang_tag_dict = {
```

Comment:
hmm, if all the keys and values are the same, why is it a `Dict` and not a `List`? I understand that you want to maintain some consistency across different multi-lingual models. If it's not a mBART-specific thing, then maybe better to store them in a list and initalize the dict with that list. Leaves smaller room for error this way.

Reply:
Yes, I was using a dict just to stay consistent with mBART. I can change the initialization for this dictionary to be from a list though